packages/volto-slate/src/blocks/Text/DefaultTextBlockEditor.jsx

```diff
@@ -246,22 +235,6 @@ export const DefaultTextBlockEditor = (props) => {
           }}
         </Dropzone>
 
-        {selected && !data.plaintext?.trim() && !disableNewBlocks && (
```

Comment:
I think it's great news that the Text block no longer has to take care of rendering the Add block button and it's instead moved to the Wrapper, as it should be.